src/java/htsjdk/samtools/CRAMFileWriter.java

```diff
@@ -46,8 +46,9 @@
 
 @SuppressWarnings("UnusedDeclaration")
 public class CRAMFileWriter extends SAMFileWriterImpl {
-    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -2;
+    private static final int REF_SEQ_INDEX_NOT_INITIALIZED = -3;
```

Comment:
I'm assuming that this change was necessary to avoid collision with the new Slice values; it looks like the refSeqIndex variable in this class now takes on values drawn from 3 different namespaces (SAMRecord, Slice and CRAMFIleWriter) that all have to be mutually exclusive. I think it would be much cleaner and safer to define the sentinel values together someplace in an enum and use a variable of that type here that is separate from the int-valued reference index.